CHANGELOG.asciidoc

```diff
@@ -17,6 +17,7 @@ https://github.com/elastic/apm-server/compare/4daa36bd5c144cf9182afc62dc8042af66
 ==== Added
 
 - Enriched data with IP and UserAgent {pull}393[393].
+- Use embedded structs instead of nested structs {pull}688[688].
```

Comment:
a little worried that putting a changelog entry in for this implies the go API is stable - is that something we want?

Reply:
you're right